nl.json

```diff
@@ -139,54 +148,127 @@
             "more-ingame-button" : "Meer Laden"
         },
         "randomiser" : {
-            "title" : "Willekeuriger",
+           "title" : "Willekeurige items",
```

Comment:
Oops, probably my fault! Of course there has to be a space before the translation key.